source/user/mcscf.rst

```diff
@@ -1,57 +1,299 @@
-.. _theory_mcscf:
+.. _user_mcscf:
 
 Multi-configuration self-consistent field (MCSCF)
 *************************************************
 
 *Modules*: :mod:`mcscf`
 
 
+Introduction
+------------
 
-Symmetry in CASSCF
-------------------
+Multiconfigurational self-consistent field (MCSCF) methods extend Hartree-Fock (HF) theory by describing the wave function as a linear combination of determinants. 
+Unlike full configuration interaction (FCI), described in :numref:`theory_ci`, complete active space (CAS) methods perform the FCI procedure on a subset of the molecular orbitals, referred to as the "active space." 
+These methods are crucial for systems that exhibit strong electron correlation, such as transition metal complexes.
+For a detailed discussion of MCSCF methods, we direct the reader to :cite:`Helgaker2013` and :cite:`esqc`.
 
-.. literalinclude:: /../examples/mcscf/21-nosymhf_then_symcasscf.py
+The MCSCF module has two main flavors: CASCI and CASSCF. 
+CASCI uses the single reference orbitals from a HF calculation and solves the CI eigenvalue problem once.
+CASSCF calculations optimize the orbitals for the set of determinants and require solving the CI eigenvalue multiple (and possibly many) times.
 
-To restart a CASSCF calculation, you need prepare either CASSCF orbitals
-or CI coefficients (not that useful unless doing a DMRG-CASSCF calculation) or
-both.  For example:
+Minimal examples for each type of calculations are shown below.
 
-.. literalinclude:: /../examples/mcscf/13-restart.py
+CASCI
+"""""
+.. code-block:: python
 
+  import pyscf
+  mol = pyscf.M(
+      atom = 'O 0 0 0; O 0 0 1.2',
+      basis = 'ccpvdz',
+      spin = 2)
+  myhf = mol.RHF().run()
+  ncas, nelecas = (6,8)
+  mycas = myhf.CASCI(ncas, nelecas).run()
 
 
- The third argument for CASCI/CASSCF is the size of CAS space; the
-fourth argument is the number of electrons.  By default, the CAS
-solver determines the alpha-electron number and beta-electron number
-based on the attribute :attr:`Mole.spin`.  In the above example, the
-number of alpha electrons is equal to the number of beta electrons,
-since the ``mol`` object is initialized with ``spin=0``.  The spin
-multiplicity of the CASSCF/CASCI solver can be changed by the fourth
-argument::
+CASSCF
+""""""
+.. code-block:: python
 
-  >>> mc = mcscf.CASSCF(m, 4, (4,2))
-  >>> print('E(CASSCF) = %.9g' % mc.kernel()[0])
-  E(CASSCF) = -149.609461
-  >>> print('S^2 = %.7f, 2S+1 = %.7f' % mcscf.spin_square(mc))
-  S^2 = 2.0000000, 2S+1 = 3.0000000
+  import pyscf
+  mol = pyscf.M(
+      atom = 'O 0 0 0; O 0 0 1.2',
+      basis = 'ccpvdz',
+      spin = 2)
+  myhf = mol.RHF().run()
+  ncas, nelecas = (6,8)
```

Comment:
Maybe it is good to show that one has to pass nelecas as a tuple to define spin?

Reply:
You're probably right, I can do that too